makslion/integreatly-operator
# MGDAPI-2489 - 3scale api error

# Description
There is a issue when creating new tenant accounts in 3scale and smtp is not configured correctly the 3scale API create the tenant account but returns an error, to avoid this from happening the smtp secret needs to be created with empty values so the 3scale API doesn't attempt to send an email after the creation of the new tenant account.

https://issues.redhat.com/browse/MGDAPI-2489

## Type of change




## Verification steps
1) Install the operator from this branch `INSTALLATION_TYPE=multitenant-managed-api make cluster/prepare/local && INSTALLATION_TYPE=multitenant-managed-api make code/run`
2) Go to the 3scale API under the master account and create a new tenant account or run this curl command below
```
curl -v  -X POST "https://master.<CLUSTER-URL>/master/api/providers.xml" -d 'access_token=<MASTER-TOKEN>&org_name=test2&username=test2&email=test2%40test2.com&password=MT'
```
3) Make sure an error is not returned from the 3scale API.
